src/test/integration/tests/Deposit_Delegate_Queue_Complete.t.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity =0.8.12;
 import "src/test/integration/IntegrationBase.t.sol";
 import "src/test/integration/User.t.sol";
 
-contract Deposit_Delegate_Queue_Complete is IntegrationBase {
+contract Integration_Deposit_Delegate_Queue_Complete is IntegrationBase {
```

Comment:
I'm fine with making this change, but had avoided adding anything extra because I forsee these names getting very long

Although I will say it's nice to be able to `forge test --match-contract Integration` to run all the integration tests.

Reply:
after some consideration i like the renaming for the reason i mentioned above :smile: